docs/rest-api/rest-notebook.md

```diff
@@ -93,7 +93,21 @@ limitations under the License.
     </tr>
     <tr>
       <td> sample JSON input </td>
-      <td><pre>{"name": "name of new notebook"}</pre></td>
+      <td><pre>
+{
+  "name": "name of new notebook", 
+  "paragraphs": [
```

Comment:
I have no good idea for now. Maybe listing sample one with 'paragraphs'  another without 'paragraphs'?